src/components/events/rbc.css

```diff
@@ -1,3 +1,23 @@
 .rbc-header:first-child {
   border-top-left-radius: 30px;
 }
+
+.rbc-header:last-child {
+  border-top-right-radius: 30px;
+}
+
+.rbc-month-row:last-child .rbc-row-bg .rbc-day-bg:first-child {
+  border-bottom-left-radius: 30px;
+}
+
+.rbc-month-row:last-child .rbc-row-bg .rbc-day-bg:last-child {
+  border-bottom-right-radius: 30px;
+}
+
+.rbc-header {
+  padding: 0 0px;
+}
+
+.rbc-month-view {
```

Comment:
keep the border and just round it out or add shadow (you can use tailwind for the shadow). the calendar blends in with the background.